engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/service/ServicePreparer.scala

```diff
@@ -0,0 +1,61 @@
+package pl.touk.nussknacker.k8s.manager.service
+
+import pl.touk.nussknacker.engine.api.{LiteStreamMetaData, ProcessVersion, RequestResponseMetaData}
+import pl.touk.nussknacker.engine.canonicalgraph.CanonicalProcess
+import pl.touk.nussknacker.k8s.manager.K8sDeploymentManager.{labelsForScenario, nussknackerInstanceNameLabel, objectNameForScenario, objectNamePrefixedWithNussknackerInstanceName, scenarioIdLabel, scenarioVersionAnnotation}
+import pl.touk.nussknacker.k8s.manager.K8sDeploymentManagerConfig
+import skuber.{ObjectMeta, Service}
+import io.circe.syntax.EncoderOps
+import pl.touk.nussknacker.engine.api.process.ProcessName
+import pl.touk.nussknacker.k8s.manager.K8sUtils.sanitizeObjectName
+import pl.touk.nussknacker.k8s.manager.service.ServicePreparer.{defaultServiceName, runtimePodTargetPort, serviceName}
+import skuber.Service.Port
+
+class ServicePreparer(config: K8sDeploymentManagerConfig) {
```

Comment:
K8sServicePreparer ? does it in general do something only with rr metadata ?

Reply:
We have DeploymentPreparer without K8s - only package determine k8s part. For now it is for rr,  but I'm a fun of mixing both approaches in the same DM so I don't wanted to make it only for rr know. But if you want I can make it rr only and we can change to more general purpose in the future when(if) both approaches will merge. WDYT?